cellprofiler/modules/_help.py

```diff
@@ -217,6 +217,13 @@
 The resulting image can also be saved with the **SaveImages** module.
 """
 
+HELP_FOR_STREL = """\
+The structuring element is the shape that will be applied in any morphological
```

Comment:
@karhohs gave what I thought was a good explanation of structuring elements, is it worth including that here?

Reply:
I liked Kyle's explanation in general, but I didn't find it very intuitive for non-mathy people (which is a BIG segment of our core audience.  I'm debating whether or not to add a figure.